build.gradle

```
@@ -46,7 +46,7 @@ neoForge {
         }
 
         data {
-        	data()
+            serverData()
```

```suggestion
            clientData()
```
This is mostly a nitpick as I do realise we don't actually make use of datagen in GB, but it's recommended that if you adapt a buildscript to still use a single data generation side, for that side to be client so that in the future assuming we add datagen we can still generate client files like lang files.

---

Make data gen client not server